services/src/main/java/za/co/sfy/lucid/sams/rest/service/TopicsService.java

```diff
@@ -0,0 +1,53 @@
+package za.co.sfy.lucid.sams.rest.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import za.co.sfy.lucid.sams.domain.ServiceStatus;
+import za.co.sfy.lucid.sams.domain.exception.LucidSamsExecutionException;
+import za.co.sfy.lucid.sams.resource.TopicsResource;
+import za.co.sfy.lucid.sams.rest.mapper.TopicsMapper;
+import za.co.sfy.lucid.sams.rest.vo.data.writer.TopicsRequests;
+import za.co.sfy.lucid.sams.rest.vo.data.writer.TopicsResponse;
+import za.co.sfy.sams.lucid.schema.Topics;
+
+@Service
+public class TopicsService {
+
+    private final TopicsResource TopicsResource;
+    private final TopicsMapper TopicsMapper;
```

Comment:
Variable naming convention violation: the first word is always lowercase and each word after that the first letter is uppercase

Please change "TopicsMapper" to "topicsMapper" and "TopicsResource" to "topicsResource"